### DISCUSSION TITLE
What is the max videos you are reliably scraping?

### DISCUSSION THREAD

**Discussion by jasonwonton** [davidteather/TikTok-Api#1106] [General]
Just wanted to get a sense of how many videos you guys are able to get in one ms_token before you need to switch it.

**Comment by calvin5walters** [comment]
Hi Jason,

Does one ms_token only allow you so many videos to be scraped?  And then you have to change it?  I have always set my token to be "ms_token".  Should I be changing it periodically?  I guess I am not clear on the function of the ms_token either.  Thanks for any help on this.

**Comment by mthirumalai2905** [comment]
am getting an error async generator can not be iterable
